### DISCUSSION TITLE
github-readme-stats is now sponsored by ▲Vercel!

### DISCUSSION THREAD

**Discussion by anuraghazra** [anuraghazra/github-readme-stats#703] [Announcement]
## Thank you ▲Vercel 

I just wanted to say huge thanks to ▲Vercel for sponsoring this project and supporting OSS in general. :heart:

#### Announcement Tweet: 
https://twitter.com/anuraghazru/status/1336958656689016834

![Powered by vercel](https://raw.githubusercontent.com/anuraghazra/github-readme-stats/master/powered-by-vercel.svg)

**Comment by ghost** [comment]
great

**Comment by ghost** [comment]
Hey people!

It's really cool to see an open source project like github-readme-stats being successful (because it makes me believe in my dreams and that's a source of inspiration, and because someday I hope to succeed doing with an open source project like: github-readme-stats too).

I still don't have time to help with this open source project, however I think this project is so amazing that it's on my to-do list to help out someday.

**Reply by rickstaa** [reply]
Thanks for the kind words! It is excellent to hear you like the OS nature of our project ❤️!

**Reply by ghost** [reply]
> Thanks for the kind words! It is excellent to hear you like the OS nature of our project ❤️!

Your words are also kind, someday I hope we can work together developing and implementing features, resolving issues.  I recently created my open project called ["all-license"](https://github.com/solarsbeans/all-license), which is the idea of generating data and statistics about software licenses. 

The github-readme-stats project is a very interesting project. So the open project: ["all-license"](https://github.com/solarsbeans/all-license) is inspired by it (github-readme-stats), although we have different goals, but we have a common idea: "readme-stats".

I must warn and emphasize that I am not promoting myself, but talking about bibliographical references in what I am inspired to do.

**Reply by anuraghazra** [reply]
Thanks for the kind words @solarsbeans, I'm glad that our small project that we started few years back helped you inspire and encourage to create your own OSS projects, that's a really great thing to see. ❤️ 

Keeping doing the great work!
